src/blog/contributors-youve-earned-a-gitpoap/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Hey Trufflesuite contributors, you’ve earned a POAP!
+hide:
+  - navigation
+---
+
+**By Kingsley Arinze**
+
+Hey Trufflesuite contributors, we’re excited to share that in collaboration with the [GitPOAP](https://gitpoap.io/) team, we will be issuing POAPs in appreciation of your awesome contributions over the years. 
+
+## What are POAPs?
+
+POAPs (Proof Of Attendance Protocol) are badges issued as NFTs (non-fungible token) to prove that the recipient/owner has carried out an action or has attended an event. They are like gold medals from the Olympics, except these exist as digital collectibles. 
+
+## Who's eligible to claim a GitPOAP?
+
+Great question. Annual contributor POAPs will be issued to those who have contributed at least once all year since the start of the following projects:
```

Comment:
I don't think the "once all year" part makes grammatical sense here. I think "one of more times since" would be better?

Reply:
agreed, updated (1833ced8)...although i assume you meant "_one or more times since_" 😊